yal/tests/test_main.py

```diff
@@ -73,6 +74,60 @@ def test_no_state_name_clashes():
     assert len(intersection) == 0, f"Common states to both apps: {intersection}"
 
 
+def test_all_states_added_to_docs():
+    m_states = set(s for s in dir(Application) if s.startswith("state_"))
+    mm_states = set(s for s in dir(MainMenuApplication) if s.startswith("state_"))
+    on_states = set(s for s in dir(OnboardingApplication) if s.startswith("state_"))
+    oo_states = set(s for s in dir(OptoutApplication) if s.startswith("state_"))
+    te_states = set(s for s in dir(TermsApplication) if s.startswith("state_"))
+    cp_states = set(
+        s for s in dir(ChangePreferencesApplication) if s.startswith("state_")
+    )
+    q_states = set(s for s in dir(QuizApplication) if s.startswith("state_"))
+    pc_states = set(s for s in dir(PleaseCallMeApplication) if s.startswith("state_"))
+    sf_states = set(s for s in dir(ServiceFinderApplication) if s.startswith("state_"))
+    aaq_states = set(s for s in dir(AaqApplication) if s.startswith("state_"))
+    fb_states = set(s for s in dir(FeedbackApplication) if s.startswith("state_"))
+    c_fb_states = set(
+        s for s in dir(ContentFeedbackSurveyApplication) if s.startswith("state_")
+    )
+    sf_s_states = set(
+        s for s in dir(ServiceFinderFeedbackSurveyApplication) if s.startswith("state_")
+    )
+    wa_fb_states = set(
+        s for s in dir(WaFbCrossoverFeedbackApplication) if s.startswith("state_")
+    )
+
+    existing_states = (
+        m_states
+        | mm_states
+        | on_states
+        | oo_states
+        | te_states
+        | cp_states
+        | q_states
+        | pc_states
+        | sf_states
+        | aaq_states
+        | fb_states
+        | c_fb_states
+        | sf_s_states
+        | wa_fb_states
+    )
```

Comment:
Should we maybe not extract this into a function? Otherwise there are two places to update this list, which makes it easy to miss.